channeldb/migrations_test.go

```diff
@@ -0,0 +1,71 @@
+package channeldb
+
+import (
+	"crypto/sha256"
+	"testing"
+)
+
+func TestPaymentStatusesMigration(t *testing.T) {
+	t.Parallel()
+
+	fakePayment := makeFakePayment()
+	paymentHash := sha256.Sum256(fakePayment.PaymentPreimage[:])
+
+	// add fake payment to test database and verify that it was created and we
```

Comment:
All comments should be complete sentences according to the contribution guidelines.

Reply:
This applies for the entire diff, so will only note it here to avoid a bunch of duplicate comments.